api/api.yaml

```diff
@@ -1,8 +1,8 @@
 openapi: 3.1.0
 info:
-  title: Self Hosted ID Platform
+  title: Self Hosted ISSUER ID
```

Comment:
Let's make a note to iterate on this title and consult with Product before release so that it's correct and consistent everywhere.